pkg/build/strategies/layered/layered.go

```diff
@@ -139,8 +150,12 @@ func (b *Layered) Build(config *api.Config) (*api.Result, error) {
 	b.config.LayeredBuild = true
 	// new image name
 	b.config.BuilderImage = newBuilderImage
-	// the scripts are inside the image
-	b.config.ScriptsURL = "image://" + filepath.Join(getDestination(config), "scripts")
+	// the scripts are inside the image, but locatnoi may vary based on -s / --scripts / --scripts-url usage
+	if len(config.ScriptsURL) > 0 {
+		b.config.ScriptsURL = "image://" + filepath.Join(getDestination(config), "scripts")
+	} else {
+		b.config.ScriptsURL = os.Getenv("STI_SCRIPTS_PATH")
```

Comment:
if there is no ./scripts directory locally, just skip settings this and leave it to default